lib/r10k/module/forge.rb

```diff
@@ -61,10 +63,9 @@ def expected_version
     @expected_version
   end
 
-
   # @return [R10K::SemVer] The version of the currently installed module
   def current_version
-    @metadata.read
+    #@metadata.read(@path + 'metadata.json') #Why would it not be already read?
```

Comment:
https://github.com/puppetlabs/r10k/commit/a20870fdd551e7b0cfb9b3c23a0cb42393c4f201 - the metadata file would not automatically be read when instantiated, which meant that the version field would be nil until that read. Not great behavior, and I think that's an artifact of how I implemented the metadata file. As long as the metadata fields will be automatically populated when we read this, we can omit the explicit read.